src/com/owncloud/android/lib/common/authentication/OwnCloudBasicCredentials.java

```diff
@@ -50,6 +54,8 @@ public OwnCloudBasicCredentials(String username, String password, boolean preemp
 
     @Override
     public void applyTo(OwnCloudClient client) {
+        AuthPolicy.registerAuthScheme(AuthState.PREEMPTIVE_AUTH_SCHEME, BasicScheme.class);
```

Comment:
OK, constant ```AuthState.PREEMPTIVE_AUTH_SCHEME``` is just "Basic", nothing to do really with preemptive mode of authentication in library; so no harm there.